### ISSUE TITLE
recipe for target 'znep.out' failed error

### ISSUE THREAD

**Issue by caobo1997** [Reference-LAPACK/lapack#474]
I am honour to learn LAPACK package,  I created the make.inc file with the example(the content is the same as make.inc.example).
But when I installed it with 'make', there was some errors: 

make[2]: Leaving directory '/home/cb/Documents/DBSR3-master/T/Libraries_03/lapack-master/TESTING/EIG'
NEP: Testing Nonsymmetric Eigenvalue Problem routines
./EIG/xeigtstz < nep.in > znep.out 2>&1
Makefile:468: recipe for target 'znep.out' failed
make[1]: *** [znep.out] Error 139
make[1]: Leaving directory '/home/cb/Documents/DBSR3-master/T/Libraries_03/lapack-master/TESTING'
Makefile:69: recipe for target 'lapack_testing' failed
make: *** [lapack_testing] Error 2

And I found in the previous issue that someone used the command 'ulimit -s unlimited' to solve this problem
(I'm very sorry that I didn't found how to cite him). But this does not solve the problem, the terminal information is:
		

                           	-->   LAPACK TESTING SUMMARY  <--
		Processing LAPACK Testing output found in the TESTING directory
SUMMARY             	       nb test run 	numerical error   	other error  
================   	===========	=================	================  
REAL             	                 1316145		0	(0.000%)	0	(0.000%)
DOUBLE PRECISION	 1316957		0	(0.000%)	0	(0.000%)
COMPLEX          	         776316		0	(0.000%)	0	(0.000%)
COMPLEX16         	         767216		1	(0.000%)	1	(0.000%)

--> ALL PRECISIONS	4176634		1	(0.000%)	1	(0.000%)

Can anyone help me?  Thank you

**Comment by martin-frbg** [comment]
Do you really still get the `recipe for target 'znep.out' failed` ? From the summary table you posted, it looks like all the tests completed now, with only two problems in the COMPLEX16 tests. (You would need to look in the file TESTING/testing_results.txt for two lines with messages that probably contain the word "failed", to find where the problem is.) Most likely your build is good and it is only some rounding error caused by compiler optimization.
What is your hardware and compiler ?

**Comment by caobo1997** [comment]
My compiler is gfortran and install on ubuntu 18.04(VMware Workstation). I check the file TESTING/testing_result.txt and find the error message:

 ZGV drivers:      1 out of   1092 tests failed to pass the threshold
 *** Error code from ZDRGEV3 =    7

**Comment by caobo1997** [comment]
> Do you really still get the `recipe for target 'znep.out' failed` ? From the summary table you posted, it looks like all the tests completed now, with only two problems in the COMPLEX16 tests. (You would need to look in the file TESTING/testing_results.txt for two lines with messages that probably contain the word "failed", to find where the problem is.) Most likely your build is good and it is only some rounding error caused by compiler optimization.
> What is your hardware and compiler ?

My compiler is gfortran and install on the Ubuntu18.04(VMware Workstation). I checked the file TEST/testing_results.txt and found the error message:

 ZGV drivers:      1 out of   1092 tests failed to pass the threshold
 *** Error code from ZDRGEV3 =    7

**Comment by weslleyspereira** [comment]
Hi.

I had a similar error on a Ubuntu 18.04.5 LTS (5.4.0-64-generic) using GNU Fortran (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0.

`make lapack_testing` returns:
```
Testing COMPLEX16          Nonsymmetric-Generalized-Eigenvalue-Problem-driver-zgd.out  ZDRGEV3: ZGGEV31 returned INFO=     7.
  ZGV drivers:      1 out of   1092 tests failed to pass the threshold
 passed: 9390
failing to pass the threshold: 1
Info Error: 1
```
and
```
			-->   LAPACK TESTING SUMMARY  <--
		Processing LAPACK Testing output found in the TESTING directory
SUMMARY             	nb test run 	numerical error   	other error  
================   	===========	=================	================  
REAL             	1316145		0	(0.000%)	0	(0.000%)	
DOUBLE PRECISION	1316957		0	(0.000%)	0	(0.000%)	
COMPLEX          	776316		0	(0.000%)	0	(0.000%)	
COMPLEX16         	689936		1	(0.000%)	1	(0.000%)	

--> ALL PRECISIONS	4099354		1	(0.000%)	1	(0.000%)
```

In the `testing_results.txt` I get:
```
 ZGV -- Complex Generalized eigenvalue problem driver
 Matrix types (see ZDRGEV3 for details): 
 Special Matrices:                       (J'=transposed Jordan block)
   1=(0,0)  2=(I,0)  3=(0,I)  4=(I,I)  5=(J',J')  6=(diag(J',I), diag(I,J'))
 Diagonal Matrices:  ( D=diag(0,1,2,...) )
   7=(D,I)   9=(large*D, small*I)  11=(large*I, small*D)  13=(large*D, large*I)
   8=(I,D)  10=(small*D, large*I)  12=(small*I, large*D)  14=(small*D, small*I)
  15=(D, reversed D)
 Matrices Rotated by Random Orthogonal Matrices U, V:
  16=Transposed Jordan Blocks             19=geometric alpha, beta=0,1
  17=arithm. alpha&beta                   20=arithmetic alpha, beta=0,1
  18=clustered alpha, beta=0,1            21=random alpha, beta=0,1
 Large & Small Matrices:
  22=(large, small)   23=(small,large)    24=(small,small)    25=(large,large)
  26=random O(1) matrices.

 Tests performed:    
 1 = max | ( b A - a B )'*l | / const.,
 2 = | |VR(i)| - 1 | / ulp,
 3 = max | ( b A - a B )*r | / const.
 4 = | |VL(i)| - 1 | / ulp,
 5 = 0 if W same no matter if r or l computed,
 6 = 0 if l same no matter if l computed,
 7 = 0 if r same no matter if r computed,

 Matrix order=   12, type=16, seed=2855,3551,2406,2393, result  1 is 4.504D+15
 ZGV drivers:      1 out of   1092 tests failed to pass the threshold
 *** Error code from ZDRGEV3 =    7```

**Comment by weslleyspereira** [comment]
@caobo1997, could you please verify if @thijssteel's solution at https://github.com/thijssteel/lapack/tree/fix-475-eigenvalue-convergence-failure solves the failing test? It worked for me.

```
			-->   LAPACK TESTING SUMMARY  <--
		Processing LAPACK Testing output found in the TESTING directory
SUMMARY             	nb test run 	numerical error   	other error  
================   	===========	=================	================  
REAL             	1316145		0	(0.000%)	0	(0.000%)	
DOUBLE PRECISION	1316957		0	(0.000%)	0	(0.000%)	
COMPLEX          	776316		0	(0.000%)	0	(0.000%)	
COMPLEX16         	777128		0	(0.000%)	0	(0.000%)	

--> ALL PRECISIONS	4186546		0	(0.000%)	0	(0.000%)
```

**Comment by thijssteel** [comment]
Wouldn't surprise me. #477 solves a pretty big problem in ZHGEQZ to the point that i'm amazed it ever worked. Definately worth a shot.

**Comment by weslleyspereira** [comment]
This issue closes with #477. Thanks @thijssteel!
